MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -149,6 +160,21 @@ - (void)testDelegateMethodsAreCalled {
   OCMVerify([delegateMock crashes:[MSCrashes sharedInstance] didFailSendingErrorReport:errorReport withError:nil]);
 }
 
+- (void)testCrashHandlerSetupDelegateMethodsAreCalled {
+
+  // If
+  id<MSCrashHandlerSetupDelegate> delegateMock = OCMProtocolMock(@protocol(MSCrashHandlerSetupDelegate));
+  [MSWrapperCrashesHelper setCrashHandlerSetupDelegate:delegateMock];
+
+  // When
+  [[MSCrashes sharedInstance] applyEnabledState:YES];
```

Comment:
You should get a partial mock instance from self.sut and use that one to avoid unexpected result when it runs simultaneously.